src/utils/actionUtils.ts

```diff
@@ -5,16 +5,43 @@
 
 import { ActionsGetJobForWorkflowRunResponseData, ActionsGetWorkflowRunResponseData } from '@octokit/types';
 import * as moment from 'moment';
-import * as path from 'path';
+import { ThemeIcon } from 'vscode';
 import { TreeItemIconPath } from 'vscode-azureextensionui';
 import { ActionWorkflowStepData, Conclusion, Status } from "../gitHubTypings";
 import { localize } from "./localize";
-import { treeUtils } from "./treeUtils";
 
 export function getActionIconPath(data: ActionWorkflowStepData | ActionsGetJobForWorkflowRunResponseData | ActionsGetWorkflowRunResponseData): TreeItemIconPath {
-    return data.conclusion !== null ?
-        treeUtils.getThemedIconPath(path.join('conclusions', ensureConclusion(data))) :
-        treeUtils.getThemedIconPath(path.join('statuses', ensureStatus(data)));
+    let id: string;
+    if (data.conclusion !== null) {
+        switch (ensureConclusion(data)) {
```

Comment:
So the only thing is that we were using icons to try to match GitHub's (the coloring, specifically).

![image](https://user-images.githubusercontent.com/5290572/114079003-7ad41800-985e-11eb-9b4e-3a566956728c.png)


![image](https://user-images.githubusercontent.com/5290572/114078597-04cfb100-985e-11eb-9705-ac61661c676e.png)

I'm not opposed to changing this, but I was wondering if there was a certain "branding" we wanted to maintain with the GitHub related icon.

Reply:
I don't think status icons fit under "branding", so yes I think we need to use VS Code icons from an accessibility/themes perspective. I would say branding is only for things like "Azure Function App" or "Azure Subscription" where it's a concept specific to that brand. In this case, "Failed" or "Skipped" is not a concept specific to GitHub